Testscripts/Linux/SRIOV-VerifyVF-Connection.sh

```diff
@@ -126,10 +126,22 @@ while [ $__iterator -le "$vf_count" ]; do
     LogMsg "Virtual function found: $vf_interface_vm_1"
 
     # Ping the remote host
-    ping -c 11 "$static_IP_2" >/dev/null 2>&1
-    if [ 0 -eq $? ]; then
-        LogMsg "Successfully pinged $VF_IP2 through $synthetic_interface_vm_1"
-    else
+    i=0
+    ping_exit_code=1
+    while [ $i -lt 4 ]
+    do
+        i=$[$i+1]
+        sleep 10
+        ping -c 11 "$static_IP_2" > "$static_IP_2.log" 2>&1
+        ping_exit_code=$?
+        LogMsg "Try the $i times to ping $VF_IP2 through $synthetic_interface_vm_1"
+        if [ $ping_exit_code == 0 ]; then
```

Comment:
For $ping_exit_code
SC2086: Double quote to prevent globbing and word splitting.